src/osmf/js/osmf_flash.js

```diff
@@ -457,51 +469,38 @@ require("../../../html5-common/js/utils/constants.js");
       this.callToFlash("destroy");
 
       // Remove the element
-      $('#'+playerId).replaceWith('');
-      _flashVideoObject=null;
-
-      // return unbound object.
-      return JFlashBridge.unbind(playerId);
-    };
-
-    // Returns the SWF instance
-    this.swf = function () {
-      return JFlashBridge.getSWF(playerId);
+      $('#'+domId).replaceWith('');
+      _video=null;
+      $(_video).remove();
+      delete flashItems[this.id];
     };
 
     // Calls a Flash method
     this.callToFlash = function (data,dataObj) {
-      if(_flashVideoObject == undefined) { 
+      if(_video == undefined) { 
         javascriptCommandQueue.push([data,dataObj]);
       }
       else
-      {      
-        if (_flashVideoObject.sendToActionScript) {
+      {
+        if (_video.sendToActionScript) {
           dataObj = typeof dataObj != 'undefined' ? dataObj : "null";
-          return _flashVideoObject.sendToActionScript(data,dataObj);
+          return _video.sendToActionScript(data,dataObj, this.id);
         } 
-        else 
+        else
         {
```

Comment:
It would be really good if consistent coding style is maintained across the file.  I am seeng if () { (curly bracket on the same line) and then else and curly bracket on the next line.  Per our coding conventions, curly bracket should be on the same line.  I am talking about many places in this file where it is being handled without consistency.